src/literals.rs

```diff
@@ -392,7 +402,7 @@ impl SingleByteSet {
 ///
 /// TODO(burntsushi): Add some amount of shifting to this.
 #[derive(Clone, Debug)]
-pub struct SingleSearch {
+pub struct MemchrSearch {
```

Comment:
I renamed this guy because I thought the old name would be confusing in the presense of multiple search algorithms focused on finding a single literal.